src/panels.py

```diff
@@ -272,6 +282,14 @@ def execute(self, context):
                 firstMeshName = os.path.splitext(mss.meshNameArray[1].basename)[0].rstrip('._0123456789')
                 seqObj.name = createUniqueName(firstMeshName + '_sequence', bpy.data.objects)
                 seqObj.mesh_sequence_settings.isImported = True
+
+                # If we import the sequence as a single mesh, the user most likey
+                # wants to export it as an alembic. Without a modifier attached,
+                # blender won't export the alembic correctly, so we add a harmless one
+                if(mss.showAsSingleMesh):
```

Comment:
I'm not sure if this is part of the PEP8 style guide, but I prefer putting one space between `if` and the condition, and only using parentheses if it will help show order of operations for complex conditionals.
So like this: `if mss.showAsSingleMesh:`
Or like this for a more complicated one: `if (thingA and thingB) or thingC:`